code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -26,6 +26,14 @@
 	weapon_weight = WEAPON_HEAVY
 
 	pb_knockback = 2
+	
+	initial_caliber = CALIBER_SHOTGUN
+	alternative_caliber = CALIBER_IMPROV_SHOTGUN
+	can_modify_ammo = TRUE
+	alternative_ammo_misfires = TRUE
+	can_misfire = FALSE
+	misfire_probability = 10 // 10% chance for your improvised shells to blow up in your face on the first fire.
```

Comment:
Just as a heads up, I think setting a baseline of 10% is actually a bad idea. I didn't give any of the original weapons higher than 0 for a reason. RNG mechanics punishing you without a real way to work around them is something I was warned against by I believe actioninja, and I agreed with his reasoning at the time. 

Currently, the two weapons that do use these mechanics start at a baseline of 0, and every shot past the first one is a risk. This makes it more reliable, but you can push your luck when the cards are down to try and get another high damage shot in. The agency is with the player as to whether they engage with those mechanics or not, while still getting some use out of the weapon when not. It's basically having your cake and eating it too.

Obviously, egg on my face for not really putting that into writing, but I believe at the time there was no move towards design docs, and I should have come back and done that.

Reply:
needs to be some kind of baseline because otherwise they can just fire, clean, fire on repeat to never have it blow up in their face

Reply:
Cleaning takes 10 seconds. This means it is never practical in an a fight to clean. You can get ONE shot. Which typically isn't enough to down on its own.